packages/apollos-api/src/data/auth/index.js

```diff
@@ -3,14 +3,13 @@ import { Auth } from '@apollosproject/data-connector-rock';
 const { schema, resolver, contextMiddleware } = Auth;
 
 class dataSource extends Auth.dataSource {
-  createUserProfile = async (props = {}) => {
+  createUserProfile = async ({ email, ...otherFields }) => {
     try {
-      const { email } = props;
-
       return await this.post('/People', {
         Email: email,
-        IsSystem: false, // Required by Rock
         Gender: 0, // Required by Rock
+        ...otherFields,
```

Comment:
@vinnyjth should this not `PascalCase` the keys? Won't they be `camcelCase`d by default?

Reply:
The GQL inputs keys are all CamelCased, so we'll be safe. 

https://github.com/ApollosProject/apollos-prototype/blob/master/packages/apollos-church-api/local.graphql#L488

Reply:
you mean `PascalCase`d, but yeah okay cool.